operator/apis/loki/v1/lokistack_types.go

```diff
@@ -838,6 +838,13 @@ type LokiStackSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tenants Configuration"
 	Tenants *TenantsSpec `json:"tenants,omitempty"`
+
+	// AdminGroups defines the list of group to be considered cluster admins by the Loki Operator.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Admin Groups"
+	AdminGroups []string `json:"adminGroups,omitempty"`
```

Comment:
This field declares configuration that is not generally applicable. In fact is only applicable for tenant modes `openshift-logging` and `openshift-network`. Therefore I suggest to wrap this withing the tenants spec, e.g.:
```yaml
spec:
  tenants:
    mode: openshift-logging
    openshiftLogging:
      adminGroups:
      - a
      - b
      - c
```